pkg/operators/clustermanager/controllers/certrotationcontroller/signer.go

```diff
@@ -0,0 +1,114 @@
+package certrotation
```

Comment:
i would put signer/cabundle into a different dir from controller

Reply:
Done. I also move the target into a new dir.